source/admin_manual/list_of_events.rst

```diff
@@ -821,70 +821,79 @@ These events apply only for connections using the ``connection API``.
 +---------------------+------------------------------------------------------+
-| port                | Target port                                          |
+| dest_host           | TCP connection's destination hostname, if known.     |
 +---------------------+------------------------------------------------------+
-| bytes_in            | Amount of data read, in bytes                        |
+| socket_path         | UNIX socket connection's path                        |
 +---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
+| remote_pid          | Remote UNIX socket server's process ID.              |
 +---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
+| remote_uid          | Remote UNIX socket server's system user ID.          |
 +---------------------+------------------------------------------------------+
 
-
-server_connection_connected
+client_connection_connected
 ---------------------------
 
-Emitted when a server connection is established.
+Emitted when a server accepts an incoming client connection.
+
+
+client_connection_disconnected
+------------------------------
+
+Emitted when a client connection is terminated.
 
 +---------------------+------------------------------------------------------+
 | Field               | Description                                          |
 +=====================+======================================================+
-| client_ip           | Source IP address                                    |
-+---------------------+------------------------------------------------------+
-| client_port         | Source port                                          |
-+---------------------+------------------------------------------------------+
-| ip                  | Target IP address                                    |
-+---------------------+------------------------------------------------------+
-| port                | Target port                                          |
-+---------------------+------------------------------------------------------+
 | bytes_in            | Amount of data read, in bytes                        |
 +---------------------+------------------------------------------------------+
 | bytes_out           | Amount of data written, in bytes                     |
 +---------------------+------------------------------------------------------+
+| reason              | Disconnection reason                                 |
++---------------------+------------------------------------------------------+
 
 
+server_connection_connected
+---------------------------
+
+Emitted when an outgoing server connection is either successfully established
+or it fails. Currently it is not possible to know which one happened.
```

Comment:
Small nit: I think it would read better if you change "it fails" -> "failed".

Reply:
I also changed is -> was. It still sounds a bit weird though (can it be thought as "successfully failed"?), but I'm not sure how to fix it better.

Reply:
Hmm, I still see the old version.

Reply:
Oh, right, because vpn is down so git push failed. I'll just wait for it to work again.

Reply:
Oh, this is github, not internal git. Just nameserver isn't working since it's pointing inside vpn. So fixable..

Reply:
I think it's fine now. Doesn't read "successfully failed" to me :)